### ISSUE TITLE
Indentation detection not working well in html.erb files

### ISSUE THREAD

**Issue by ghost** [schorfES/grunt-lintspaces#15]
I’ve setup lintspaces with the following:

``` javascript
lintspaces: {
            html: {
                src: [
                    '**/*.html.erb'
                ],
                options: {
                    newline: true,
                    trailingspaces: true,
                    indentation: 'spaces',
                    spaces: 2
                }
            },
        }
}
```

but when I run it, it does not detect any problems even though all my html.erb files are indented with 4 spaces instead of 2. I’m guessing that’s because 2 is a mulltiple of 4? Any way around that?

The newline detection does work though...
